tests/actors/rust/wrpc-ponger-component/wit/deps.toml

```diff
@@ -0,0 +1,4 @@
+http = "https://github.com/WebAssembly/wasi-http/archive/v0.2.0.tar.gz"
+keyvalue = "https://github.com/WebAssembly/wasi-keyvalue/archive/main.tar.gz"
+logging = "https://github.com/WebAssembly/wasi-logging/archive/main.tar.gz"
```

Comment:
probably don't need these, though maybe it'd be nice to show logging just as an example...